rcaracaus/pledge-vegan-node
# 2-node-detail: Clean URLs

Steps to test:
git checkout master
git pull origin master
gco -b 2-node-detail--clean-urls 
git pull origin 2-node-detail--clean-urls 

*Go to http://localhost:3000/silverback-gorilla-shatters-glass-little-girl-beats-her-chest and make sure it loads properly
